sources/engine/Stride.Assets.Tests2/Stride.Assets.Tests2.csproj

```diff
@@ -12,7 +12,6 @@
   <ItemGroup>
     <PackageReference Include="xunit" Version="2.4.2-pre.12" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
-    <PackageReference Include="TeamCity.VSTest.TestAdapter" Version="1.0.32" />
```

Comment:
We should verify this test still runs fine on TeamCity - I'm not really sure what this adapter is meant to do here

Reply:
I'm afraid this duplicate package reference was also intentional [(see here)](https://github.com/stride3d/stride/commit/c21cf0224153c8d1a04a9bab6e9d37f0ea2f19f0#diff-c7b44c06ee0a39a3aa90ea0467d9fde7415f6f801ba2d45c1fe54174b1f46b78). I will undo this also for now.